### ISSUE TITLE
Improve payment totals in admin

### ISSUE THREAD

**Issue by elliott-stocks** [easydigitaldownloads/Easy-Digital-Downloads#2625]
I think the display of the "Payment Totals" lacks detail, it doesn't really give the shop owner quick information about the order.

I think it should at least have the following -

1. Subtotal
2. Discount
3. Fees
4. Tax
5. Total Price

For example

![order-totals-example](https://cloud.githubusercontent.com/assets/1441538/4305196/4a8c6794-3e77-11e4-8d61-fbde1eda64de.PNG)

It currently shows which discount code has been used, but not the amount of discount given. I think discount should show the code and the amount of discount applied.

On the order details page everything is referred to as "payment", though technically it isn't a payment until the order has been confirmed (if it's abandoned for example). What are your thoughts on changing Payment to Order instead?

Possibly related #2518

**Comment by cklosowski** [comment]
Going to jump into this, are we thinking of splitting all 'payment total' items out of the 'update payment' metabox?

Currently ```Update Payment``` contains:
* Status
* Date
* Time
* Discount Code
* Tax
* Total Price

Updating to this would probably mean we have two boxes from one:
```Update Payment```
* Status
* Date
* Time

* Subtotal
* Discount (Code and then Amount)
* Fees
* Tax (Editable)
* Total (Editable)

Is this looking right?

**Comment by pippinsplugins** [comment]
I like the idea of separating it and the separation looks good.

**Comment by sdavis2702** [comment]
I agree that looks good.

**Comment by cklosowski** [comment]
:+1: thanks guys

**Comment by cklosowski** [comment]
Ok, after some discussions with @pippinsplugins we're going to  move this to 2.4. We're going to need to get a payment_meta item for the discount amount, and also have it updated when a payment is updated. On top of that, an upgrade routine is also necessary.

Since this is a nice to have, we're going to punt for the sake of keeping 2.3 timely.

**Comment by cklosowski** [comment]
I'd like to open this topic up for a reorganization of the data on the payment details if we can. I think after #2518, this one is a bit stale, since the sidebar has been reorganized completely.

@elliott-stocks, I'm going to open a new issue, linking back to this one, so we can start re-envisioning what a payments screen should look like. Plus we're pretty close to 2.4 and I think this will be an involved ticket so I don't want to rush it.
